src/Symfony/Component/Form/Tests/Extension/DataCollector/FormDataCollectorTest.php

```diff
@@ -712,20 +567,18 @@ public function testReset()
     {
         $form = $this->createForm('my_form');
 
-        $this->dataExtractor->expects($this->any())
-            ->method('extractConfiguration')
-            ->willReturn([]);
-        $this->dataExtractor->expects($this->any())
-            ->method('extractDefaultData')
-            ->willReturn([]);
-        $this->dataExtractor->expects($this->any())
-            ->method('extractSubmittedData')
-            ->with($form)
-            ->willReturn(['errors' => ['baz']]);
-
         $this->dataCollector->buildPreliminaryFormTree($form);
         $this->dataCollector->collectSubmittedData($form);
 
+        $this->assertNotSame(
```

Comment:
Is there something we can expect here? `assertNotSame` with two arrays could mean anything and don't seem to bring any value.

Reply:
I just wanted to make sure, that there actually is any collected data so that calling `reset()` afterwards actually has an effect and that the following assertion isn't passing by luck.

Reply:
OK, so what about something like `assertGreaterThan(0, $this->dataCollector->getData()['forms'])`?

Reply:
Hm yeah, might make sense. I am going to send a follow-up PR later if you haven't done that in the meantime. :)

Reply:
see #46096